download/invoke.yml

```diff
@@ -1,3 +1,12 @@
-data_dir: /data
+# Ignore files cache.
+force_downloads: False
 
-oa_base_url: https://results.openaddresses.io/latest/run/
+# Directory were to keep files that need to be cached for addresses download.
+# Note that if it is not inside a volume, the cache will probably be lost at
+# each run.
+cache_dir: /tmp/cache
+
+tmp_dir: /tmp
+
+# URL for the zip archive containing all data.
+oa_url: https://data.openaddresses.io/openaddr-collected-europe.zip
```

Comment:
This URL does not seem to be configurable from the top-level invoke configuration.  
For example, we could want to download the planet file instead.

Reply:
I moved this piece of configuration in the top-level `invoke.yml`, it seems indeed cleaner to have everything in the same place :)